src/lib/comms/sol-mqtt.c

```diff
@@ -0,0 +1,579 @@
+/*
```

Comment:
Yeah, that was its name at first, but I got a feedback that we should move to a different name after we got a second implementation. I'm ok with the rename though.